celery/app/defaults.py

```diff
@@ -95,6 +95,8 @@ def __repr__(self):
         heartbeat=Option(120, type='int'),
         heartbeat_checkrate=Option(3.0, type='int'),
         login_method=Option(None, type='string'),
+        native_delayed_delivery=Option(False, type='bool'),
+        native_delayed_delivery_queue_type=Option(default='classic', type='string'),
```

Comment:
My client wants to use this before migrating to quorum queues so the default is set to classic.

Reply:
> My client wants to use this before migrating to quorum queues so the default is set to classic.

I understand, but why does it make sense?
It makes sense to me to be default `quorum` because that’s the only flow it is used.
If not, you force the user to 1) be aware of this flag and 2) set it.
Instead, setting it to `quorum` gives an automatic behavior, which is why `classic` doesn’t make sense to me.

You may bring this up with your client to ask for clarifications, but we want to reach an agreement based on what’s best for the code.

Reply:
In a busy production environment, using quorum queues for all the native delayed delivery queues may overload RabbitMQ.
In addition, my client wants to test this feature without using quorum queues at first.

Reply:
I changed the default to quorum.